packages/tools/kadena-cli/src/wallets/commands/walletsChangeWalletPassword.ts

```diff
@@ -82,10 +82,10 @@ export const createChangeWalletPasswordCommand: (
   program: Command,
   version: string,
 ) => void = createCommand(
-  'change-wallet-password',
+  'change-password',
   'Update the password for your wallet',
   [
-    globalOptions.keyWalletSelect(),
+    globalOptions.walletWalletSelect(),
```

Comment:
renaming process became a bit clumsy, and it seems that all instances of `keyWallet` were inadvertently changed to `walletWallet` even in the subsequent files

Reply:
Actually it should have all be renamed but walletWallet is kinda messy will make it walletName